src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1080,6 +1096,10 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
               rightComponent={ReadOnlyAsset}
               formControlProps={formControlProps}
               onChange={handleAddLiquidityInputChange}
+              handleIsInputtingFiatSellAmountChange={
+                isRune ? handleToggleRuneIsFiat : handleTogglePoolAssetIsFiat
+              }
```

Comment:
also here, might be able to just consume the toggles directly without wrapping?

Reply:
Ditto re: arity - https://github.com/shapeshift/web/pull/6469/commits/b7dc9b7923eb43e6e0f7b33063148c6e784a5b41

Reply:
Absolutely - agreed the name should be more generic in the shared component! 43d2b22e2eae3191332e33f419b282bcb16050db